### PR TITLE
[7.67.x][BXMSPROD-2057] Updating references to the new kie-ci actions folder

### PR DESCRIPTION
**Pull request by bacciotti**
Related PR:
- https://github.com/kiegroup/drools/pull/5196

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -24,18 +24,19 @@ jobs:
...
+        uses: kiegroup/kie-ci/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/build-chain@main
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         with:
+          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml
```
Comment (lampajr) [inline .github/workflows/pull_request.yml]: Drools `7.67.x` is related to RHPAM, therefore we should take definition file from `droolsjbpm-build-bootstrap` to keep it exactly as it was.
```suggestion
          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/droolsjbpm-build-bootstrap/${BRANCH:main}/.ci/pull-request-config.yaml
```

**Review by lampajr — CHANGES_REQUESTED** [review]

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (Ginxo) [reply .github/workflows/pull_request.yml]: I agree @lampajr

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

**Review by lampajr — APPROVED** [review]

**Review by Ginxo — APPROVED** [review]
LGTM

**Review by mariofusco — APPROVED** [review]

**Pull request merged**